org.eclipse.transformer/src/main/java/org/eclipse/transformer/util/SignatureUtils.java

```diff
@@ -184,11 +192,11 @@ public static Stream<String> keyStream(String key, String wildcard) {
 		return StreamSupport.stream(spliterator, false);
 	}
 
-	static class KeySpliterator extends AbstractSpliterator<String> {
+	public static class KeySpliterator extends AbstractSpliterator<String> {
```

Comment:
Not sure this should be public API.

Reply:
Fixed.  Changed to 'protected', but could also be 'private'.